test/e2e/suite/scenario/crl.go

```diff
@@ -0,0 +1,232 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scenario_test
```

Comment:
One more test case that could happen a lot in reality : `the crl in the cache expired and a roundtrip is required.` 
Do we have a test case that covers this scenario?

Reply:
How's this be a unit test? It's an E2E scenario, we need to cover those failure paths when doing e2e test.

Reply:
Actually, this is an interesting test case. I will try to add it.

Reply:
Updated.